### ISSUE TITLE
Run-ALValidation - InstallApps/SortByDependencies

### ISSUE THREAD

**Issue by waldo1001** [microsoft/navcontainerhelper#2008]
**Describe the issue**
When I depend on a 3rd party application, that has split their product into multiple extensions, it's important to install these extension in the right order.

So I think that this piece should also include the `Sort-AppFilesByDependencies`
![image](https://user-images.githubusercontent.com/12088142/126774771-2fe373b3-0806-45f7-822e-66f131a8a6af.png)

So .. something in the sorts of:
```powershell
try {
    Sort-AppFilesByDependencies -containerName $containerName -appFiles @(CopyAppFilesToFolder -appFiles $installApps -folder $appsFolder) -WarningAction SilentlyContinue | ForEach-Object {
        $Parameters = @{
            "containerName" = $containerName
            "tenant" = $tenant
            "credential" = $credential
            "appFile" = $_
            "skipVerification" = $true
            "sync" = $true
            "install" = $true
        }
        try {
            Invoke-Command -ScriptBlock $PublishBcContainerApp -ArgumentList $Parameters
        }
        catch {
            $error = "Unable to install app $([System.IO.Path]::GetFileName($parameters.appFile)) on container based on $($artifactUrl.Split('?')[0]). You can try to re-run with -skipUpgrade.`nError is: $($_.Exception.Message)"
            $validationResult += $error
            Write-Host -ForegroundColor Red $error
        }
    }
}
finally {
    Remove-Item -Path $appsFolder -Recurse -Force
}

} | ForEach-Object { Write-Host -ForegroundColor Yellow "`nInstalling apps took $([int]$_.TotalSeconds) seconds" }
```

**Comment by freddydk** [comment]
I actually think this:
```
    $installApps | ForEach-Object{
        $Parameters = @{
            "containerName" = $containerName
            "tenant" = $tenant
            "credential" = $credential
            "appFile" = $_
            "skipVerification" = $true
            "sync" = $true
            "install" = $true
        }
        Invoke-Command -ScriptBlock $PublishBcContainerApp -ArgumentList $Parameters
    }
```
can be replaced with this:
```
    $Parameters = @{
        "containerName" = $containerName
        "tenant" = $tenant
        "credential" = $credential
        "appFile" = $installApps
        "skipVerification" = $true
        "sync" = $true
        "install" = $true
    }
    Invoke-Command -ScriptBlock $PublishBcContainerApp -ArgumentList $Parameters
```
as publish does the sort inside

**Comment by freddydk** [comment]
I will test this next week when returning from Vacation

**Comment by freddydk** [comment]
Shipped in 2.0.15
